dev/tests/integration/testsuite/Magento/Framework/Image/Adapter/InterfaceTest.php

```diff
@@ -115,9 +115,12 @@ protected function _getFixture($pattern)
      */
     protected function _isFormatSupported($image, $adapter)
     {
+        if (!$image || !file_exists($image)) {
+            return false;
+        }
         $data = pathinfo($image);
-        $supportedTypes = $adapter->getSupportedFormats();
-        return $image && file_exists($image) && in_array(strtolower($data['extension']), $supportedTypes);
+
+        return isset($data['extension']) && in_array(strtolower($data['extension']), $adapter->getSupportedFormats());
```

Comment:
Hello @anzin 
Could you please check my suggestion

```suggestion
        $supportedTypes = $adapter->getSupportedFormats();
        
        return isset($data['extension']) && in_array(strtolower($data['extension']), $supportedTypes);
```

Reply:
Code fixed by recommendation, please check if every think is well.

Thank you.